autosklearn/ensemble_builder.py

```diff
@@ -250,9 +248,7 @@ def ensemble_builder_process(
             )
 
         # Don't take resources while waiting
-        dask.distributed.secede()
         time.sleep(sleep_duration)
```

Comment:
I think this part could be slightly reworked. Previously, if the ensemble script constructed a new ensemble, it wouldn't sleep. It would only sleep if there were no new predictions available. So I guess if you could add an additional flag to the fit_and_return_ensemble that indicates whether an ensemble was constructed, that would bring back the original behavior.